stroyprombeton/tests/tests_views.py

```diff
@@ -1079,14 +1055,111 @@ def test_product_images(self):
         product = models.Product.objects.get(id=PRODUCT_WITH_IMAGE)
         series = models.Series.objects.first()
         product.options.update(series=series)
-        response = self.client.get(series.url)
+        response = self.client.get(series.url)  # Ignore CPDBear
         image = response.context['product_images'][110]
         self.assertTrue(image)
         self.assertTrue(image.image.url)
 
     def test_product_image_button(self):
         """Series page should contain button to open existing product image."""
         # product with image
+        product = models.Product.objects.get(id=PRODUCT_WITH_IMAGE)  # Ignore CPDBear
+        response = self.client.get(product.category.url)
+        self.assertContains(response, 'table-photo-ico')
+
+
+@tag('fast', 'catalog')
+class SectionFirst(BaseCatalogTestCase):
+    fixtures = ['dump.json']
+
+    def test_200_ok(self):
+        section = models.Section.objects.create(
+            name='First section',
+            slug='fist',
+            page=ModelPage.objects.create(
+                name='First section',
+            ),
+        )
+        models.Product.objects.active().update(section=section)
+        assert section.page
```

Comment:
It looks weird, because we create this instance of the `Page` a few lines above

Reply:
sorry, it was debugging row